setup_wizard/character_rig_setup/rig_script.py

```diff
@@ -2764,8 +2772,10 @@ def bone_to_layer(bone, layer, collection):
                 else:
                     bpy.context.object.data.collections[collection].assign(bpy.context.object.data.bones[bone])
                     bpy.context.object.data.collections["Other"].unassign(bpy.context.object.data.bones[bone])
+                    if second_coll != "None":

```

Comment:
Food for thought, could've probably went with using a default arg of `None`
then you can set up the conditional like so:
```py
if second_coll:
```